pandas/_libs/tslibs/parsing.pyx

```diff
@@ -238,6 +238,17 @@ cdef inline bint does_string_look_like_time(str parse_string):
 
     return 0 <= hour <= 23 and 0 <= minute <= 59
 
+from pandas.util._exceptions import find_stack_level
```

Comment:
is there an option that avoids importing from outside libs here?

Reply:
yup, could set it explicitly each time - I'll do that once I've sorted out the tests